crates/core/src/db/datastore/locking_tx_datastore/state_view.rs

```diff
@@ -324,40 +318,38 @@ pub struct IndexSeekIterMutTxId<'a> {
     pub(crate) num_committed_rows_fetched: u64,
 }
 
-#[cfg(feature = "metrics")]
 impl Drop for IndexSeekIterMutTxId<'_> {
```

Comment:
Are the analogous changes for `IndexSeekIterMutTxId` also correct?

Reply:
this looks good.